fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -70,6 +69,9 @@
     private boolean isAllowNull;
     @SerializedName(value = "defaultValue")
     private String defaultValue;
+    // this handle function like now() or simple expression
+    @SerializedName(value = "defaultExpr")
+    private String defaultExpr;
```

Comment:
I think the default value should be encapsulated into another class.
This original defaultValue is a special case which is a literal expression.
Because more functions will supported later, we should do some abstract now to avoid changing later.